composer.json

```
@@ -70,32 +71,38 @@
         ],
         "ci:php:fixer": "\"./.phive/php-cs-fixer\" --config=config/php-cs-fixer.php fix --dry-run -v --show-progress=dots --diff bin src tests",
         "ci:php:stan": "phpstan --no-progress --configuration=config/phpstan.neon",
+        "ci:php:rector": "\"./vendor/bin/rector\" --no-progress-bar --no-diffs --dry-run --config=config/rector.php",
```

```suggestion
        "ci:php:rector": "rector --no-progress-bar --no-diffs --dry-run --config=config/rector.php",
```

Also, I think that having the diff might be helpful to the human reader. What do you think?

---

Fixed.